packages/ecmascript/transpilation-tests.js

```diff
@@ -40,7 +40,7 @@ Tinytest.add("ecmascript - transpilation - class methods", (test) => {
   test.isTrue(contains(output, 'Foo.staticMethod = function staticMethod('));
   test.isTrue(contains(output,
                        '.prototypeMethod = function prototypeMethod('));
-  test.isTrue(contains(output, '[computedMethod] = function ('));
+  test.isTrue(contains(output, '[_computedMethod] = function ('));
```

Comment:
Was this just breaking tests and not actual code?

Reply:
As far as I can see yes. I will look if I can find something relevant in the changelogs, but from what I have been running the code and other tests there weren't any adverse effects.
:thinking: Not even as far as I have tested in production build.